Dockerfile.armhf

```diff
@@ -164,10 +164,10 @@ RUN ln -sv $PWD/contrib/completion/bash/docker /etc/bash_completion.d/docker
 COPY contrib/download-frozen-image-v2.sh /go/src/github.com/docker/docker/contrib/
 RUN ./contrib/download-frozen-image-v2.sh /docker-frozen-images \
 	armhf/buildpack-deps:jessie@sha256:ca6cce8e5bf5c952129889b5cc15cd6aa8d995d77e55e3749bbaadae50e476cb \
```

Comment:
buildpack-deps:jessie and debian:jessie for armhf, ppc64le, and s390x have all been updated to have the right apt version so you can update these too